templates/tease-search.twig

```diff
@@ -90,6 +90,9 @@
 			</div>
 
 			<p class="search-result-item-content">{{ post.post_excerpt|default(post.post_content)|excerpt( 25 )|raw }}</p>
+			{% if user.can('edit_posts') %}
```

Comment:
Does this link works with load more button results?

Reply:
It does not currently, good catch! :100: